samples/tab-adaptive-cards/csharp/TabWithAdpativeCardFlow/Bots/ActivityBot.cs

```diff
@@ -0,0 +1,139 @@
+using System.Threading.Tasks;
+
+namespace TabWithAdpativeCardFlow.Bots
+{
+    public class ActivityBot : TeamsActivityHandler
+    {
+        protected override async Task<InvokeResponse> OnInvokeActivityAsync(ITurnContext<IInvokeActivity> turnContext, CancellationToken cancellationToken)
+        {
+            if (turnContext.Activity.Name == "tab/fetch")
+            {
+                return CreateInvokeResponse(new TabResponse
+                {
+                    Tab = new TabResponsePayload
+                    {
+                        Type = "continue",
+                        Value = new TabResponseCards
+                        {
+                            Cards = new List<TabResponseCard>
+                            {
+                                new TabResponseCard
+                                {
+                                    Card = GetAdaptiveCard1()
+                                },
+                                new TabResponseCard
+                                {
+                                    Card = GetAdaptiveCard2()
+                                },
+                            },
+                        },
+                    },
+                });
+            }
+            else if (turnContext.Activity.Name == "tab/submit")
+            {
+                return CreateInvokeResponse(new TabResponse
+                {
+                    Tab = new TabResponsePayload
+                    {
+                        Type = "continue",
+                        Value = new TabResponseCards
+                        {
+                            Cards = new List<TabResponseCard>
+                            {
+                                new TabResponseCard
+                                {
+                                    Card = GetAdaptiveCard2()
+                                },
+                            },
+                        },
+                    },
+                });
+            }
+
+            return null;
+        }
+
+        private AdaptiveCard GetAdaptiveCard1()
+        {
+            AdaptiveCard card = new AdaptiveCard(new AdaptiveSchemaVersion("1.2"))
+            {
+                Body = new List<AdaptiveElement>
+                {
+                    new AdaptiveTextBlock
+                    {
+                        Text = "Test the tab view with Adaptive card",
+                        Weight = AdaptiveTextWeight.Bolder,
+                    },
+                },
+
+                Actions = new List<AdaptiveAction>
+                {
+                    new AdaptiveSubmitAction
+                    {
+                        Title = "Action 1",
+                    },
+                    new AdaptiveSubmitAction
+                    {
+                        Title = "Action 2",
+                    },
+                },
+            };
+
+            return card;
+        }
+
+        private AdaptiveCard GetAdaptiveCard2()
```

Comment:
Please add summary & provide proper card name.

Reply:
Updated with sampleAdaptiveCard and added summary